lib/components/Sequencer/index.jsx

```diff
@@ -17,6 +17,7 @@ export class Sequencer extends Component {
           steps:[true,false,false,false,true,false,false,false,true,false,false,false,true,false,false,false],
           sound:{
            source : 'noise',
+           volume: 0,
```

Comment:
Does this defaulting to 0 mean the user has to turn it up to hear it initially? If so should it default to 50 or something like that?